### PR TITLE
Move to JSONB instead of MongoDB

### PR DESCRIPTION
**Pull request by ch0k0bn**

### CODE REVIEW

File: frontend/controllers/scanctrl.py [hunk 1]
```diff
@@ -420,20 +419,15 @@ def set_result(scanid, file_hash, probe, result):
...
+            # fill ProbeResult with probe raw results
+            pr.doc = sanitized_res
             pr.status = sanitized_res.get('status', None)
             s_type = sanitized_res.get('type', None)
             pr.type = IrmaProbeType.normalize(s_type)
             pr.update(session=session)
             probedone = []
             for pr in fw.probe_results:
-                if pr.nosql_id is not None:
+                if pr.doc is not None:
```
Comment (guillaumededrie) [inline frontend/controllers/scanctrl.py]: I'm not sure, but as pr.doc is a static variable, will it be more efficient to have:
```python
    if pr.doc is not None:
        probedone = [pr.name for pr in fw.probe_results]
    […]
```

Comment (guillaumededrie) [inline frontend/controllers/scanctrl.py]: my comment is for from line [435](https://github.com/quarkslab/irma-frontend/pull/126/files#diff-0a24ed5d3f543bec8dc62ab60600680cL435) to [437](https://github.com/quarkslab/irma-frontend/pull/126/files#diff-0a24ed5d3f543bec8dc62ab60600680cL437) included

Comment (ch0k0bn) [inline frontend/controllers/scanctrl.py]: its pr.doc.name and you should handle the case where pr.doc is None, we could use list comprehension but will be
more complex to read imo

**Comment by guillaumededrie** [comment]
:+1:

**Pull request merged**